policy/modules/roles/sysadm.if

```diff
@@ -236,3 +236,22 @@ interface(`sysadm_rw_pipes',`
 
 	allow $1 sysadm_t:fifo_file rw_fifo_file_perms;
 ')
+
+########################################
+## <summary>
+##  Allow sysadm_t to read/write a inherited pipes
+##  from another domain.
+## </summary>
+## <param name="domain">
+##  <summary>
+##   Domain of the inherited pipe.
+##  </summary>
+## </param>
+#
+interface(`sysadm_rw_inherited_pipe_from', `
+	gen_require(`
+		type sysadm_t;
+	')
+
+	allow sysadm_t $1:fifo_file rw_inherited_fifo_file_perms;
+')
```

Comment:
I'm not completely convinced that we want to have this in general. If we do, it would be better to add this onto the `sysadm_role_change()` interface by adding the source domain as a parameter. Maybe the the fifo_file rule should be in a conditional block (boolean) in the interface too.